python/pfs/drp/stella/fitBroadbandSED.py

```diff
@@ -0,0 +1,135 @@
+from lsst.pex.config import Config, Field, DictField
+from lsst.pipe.base import Task
+from lsst.utils import getPackageDir
+
+import numpy
+import numpy.lib.recfunctions
+import astropy.io.fits
+
+import os
+
+
+class FitBroadbandSEDConfig(Config):
+    """Configuration for FitBroadbandSEDTask
+    """
+
+    fluxLibraryPath = Field(
+        dtype=str,
+        default=os.path.join(getPackageDir("drp_pfs_data"), "fluxCalibration",
+                             "syntheticFlux_withInterp_ps_HSCPS1GaiaSDSS.fits"),
+        doc="Synthetic photometry table"
+    )
+
+    filterAliases = DictField(
+        keytype=str, itemtype=str,
+        default={
+            "g": "HSCg", "r": "HSCr", "r2": "HSCr2", "i": "HSCi", "i2": "HSCi2",
+            "z": "HSCz", "y": "HSCy"
+        },
+        doc="Conversion table from AFW's filter names to those used by `fluxLibrary`"
```

Comment:
I don't think we need to worry about AFW's filter names, and there are a lot of changes to AFW's filter handling coming soon so we should avoid it. All we need to worry about are the filter names in the pfsConfig.